src/data.js

```diff
@@ -3,7 +3,10 @@ let movies = [
       "title": "The Shawshank Redemption",
       "year": 1994,
       "director": "Frank Darabont",
-      "duration": "2h 22min",
+      "duratiofunction turnHoursToMinutes(movies){ 
+        // let hoursArr = movies[0].duration.split(" ");   
+      let newArr=movies[0].duration.split(" ");
```

Comment:
you were having some syntax errors because you added by mistake this in data.js :)